src/applications/mhv/secure-messaging/components/MessageList/MessageList.jsx

```diff
@@ -194,7 +194,7 @@ const MessageList = props => {
       <div className="vads-u-padding-y--1 vads-l-row vads-u-margin-top--2 vads-u-border-top--1px vads-u-border-bottom--1px vads-u-border-color--gray-light">
         Displaying {displayNums[0]}
         &#8211;
-        {displayNums[1]} of {totalEntries} conversations
+        {displayNums[1]} of {totalEntries} messages
```

Comment:
You might want to check with UCD on this. It's not imperative to change now but in the phase 0 mocks changed the verbiage to 'conversations'.  If you decide to update it you may need to update some cypress & unit tests to pass.

Reply:
Hey yeah, i checked with Joel and Oleksii last week on this change, however I was not able to see the cypress test affected by this, would you happen to know any?